### DISCUSSION TITLE
不具合: 記事、質問を検索で100ページを超えると Not Found 表記になる

### DISCUSSION THREAD

**Discussion by 101ta28** [increments/qiita-discussions#787] [要望・アイデア・不具合]
「記事、質問を検索」の検索欄に関する不具合です。(仕様でしたら申し訳ないです)
検索結果が100ページを超える(101ページ以降)が404 Not Found表記になり、検索ができないです。

例: 「Qiita」というワードで検索した場合
https://qiita.com/search?q=Qiita&sort=&stocked=&page=100 は検索結果が表示されます。
https://qiita.com/search?q=Qiita&sort=&stocked=&page=101 は 404 Not Found になります。

**Comment by tanaka-asami415** [comment]
@101ta28 
ご不便をおかけして申し訳ありません🙏

検索システムのパフォーマンスの都合上、現在100ページまでに制限しています。
（101ページ以上は見れない事が伝わるような修正をする予定です🙇）

検索結果が100ページ以内になるように、検索オプション等で絞り込みをしていただけると幸いです。
https://help.qiita.com/ja/articles/qiita-search-options

**Reply by 101ta28** [reply]
制限をかけている旨、承知しました。
ご返信ありがとうございます。この件はクローズさせていただきます。

**Reply by P-SiZK** [reply]
@101ta28 
お待たせしました 🙇 

100 ページを超えて検索した際の表示内容を変更しました！
下記の画像のように、100 ページまでの制限があることが分かるようにしました。

![](https://github.com/increments/qiita-discussions/assets/54477535/017f1eda-a1a7-41e5-9722-5673063c7007)
